libs/shared/README.md

```diff
@@ -0,0 +1,14 @@
+# shared
```

Comment:
remove this file or make it proper, so write what should be in the shared lib

Reply:
removed